packages/bsp/sunxi-next-h3-hdmi/asound.conf

```diff
@@ -0,0 +1,10 @@
+ pcm.!default {
```

Comment:
https://github.com/armbian/build/tree/master/packages/blobs/asound.state

Reply:
Dear @igorpecovnik I didn't get that part. I shoud the content of asound.conf to the https://github.com/armbian/build/tree/master/packages/blobs/asound.state/asound.state.sunxi-next ?